crates/bevy_text/src/lib.rs

```diff
@@ -47,10 +47,12 @@ impl Plugin for TextPlugin {
             .register_type::<HorizontalAlign>()
             .init_asset_loader::<FontLoader>()
             .insert_resource(DefaultTextPipeline::default())
+            .init_resource::<DefaultFont>()
             .add_system_to_stage(
                 CoreStage::PostUpdate,
                 update_text2d_layout.after(ModifiesWindows),
-            );
+            )
+            .add_system(load_font);
```

Comment:
Are there any system-ordering issues we should be careful about here?

Reply:
Not really, I guess it could make sense to run it in PostUpdate so it can run in the same frame as when the Text was added but it will work either way